fdbserver/workloads/BlobGranuleRangesWorkload.actor.cpp

```diff
@@ -300,6 +301,9 @@ struct BlobGranuleRangesWorkload : TestWorkload {
 	}
 
 	ACTOR Future<bool> _check(Database cx, BlobGranuleRangesWorkload* self) {
+		if (deterministicRandom()->coinflip()) {
+			cx->internal = IsInternal::False;
```

Comment:
Yes I agree, I was initially confused why this test wasn't reproducing the error until i did this. I think because it runs as an fdb server process this makes sense.
We must be doing something like this in at least the authz tests i'd think?